test/data/ansible/check-minimal.yaml

```diff
@@ -4,6 +4,7 @@
   vars:
     total_counter: "0"
     failed_counter: "0"
+    download_node: ""
```

Comment:
nitpick: `hostname` or `url` would IMO make more sense than `node`, but this is not a blocker from my PoV.

Reply:
cc @henrywang